terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverOperations.java

```diff
@@ -55,7 +55,7 @@ public void setDefaultTimeoutForImplicitlyWait(
      */
     public boolean exists(By by) {
         webDriver.findElement(By.tagName("body"));
-        setTimeoutForImplicitlyWait(0, TimeUnit.SECONDS);
+        setTimeoutForImplicitlyWait(Duration.ofSeconds(0));
```

Comment:
`Duration.Zero` is not fine?

Reply:
Use the Duration.Zero provided.

Reply:
Fixed a20f7a7c18f13432b4cbb129a57a352704932c3b in https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/986